tests/test_errors.py

```diff
@@ -664,6 +690,20 @@ def test_raise_validation_error_custom():
     ]
 
 
+@pytest.mark.parametrize(
+    'msg,result_msg', [('my custom error', 'my custom error'), ('my custom error {foo}', "my custom error {'bar': []}")]
+)
+def test_raise_validation_error_custom_nested_ctx(msg: str, result_msg: str):
+    custom_error = PydanticCustomError('my_error', msg, {'foo': {'bar': []}})
+    with pytest.raises(ValidationError) as exc_info:
+        raise ValidationError.from_exception_data('Foobar', [{'type': custom_error, 'input': 'x'}])
+
+    # insert_assert(exc_info.value.errors(include_url=False))
+    assert exc_info.value.errors(include_url=False) == [
+        {'type': 'my_error', 'loc': (), 'msg': result_msg, 'input': 'x', 'ctx': {'foo': {'bar': []}}}
+    ]
```

Comment:
Can we also see the output for `ValueError.json()`?

Reply:
Added the check